src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/Analyzers/ConvertToLibraryImportFixer.cs

```diff
@@ -70,9 +70,9 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
 
             foreach (Diagnostic diagnostic in context.Diagnostics)
             {
-                if (!bool.Parse(diagnostic.Properties[ConvertToLibraryImportAnalyzer.ExactSpelling]))
+                if (!bool.Parse(diagnostic.Properties[ConvertToLibraryImportAnalyzer.ExactSpelling]!))
                 {
-                    CharSet charSet = (CharSet)Enum.Parse(typeof(CharSet), diagnostic.Properties[ConvertToLibraryImportAnalyzer.CharSet]);
+                    CharSet charSet = (CharSet)Enum.Parse(typeof(CharSet), diagnostic.Properties[ConvertToLibraryImportAnalyzer.CharSet]!);
```

Comment:
A lot of files in this PR don't look related, and have merge conflicts.  Presumably there's a misplaced commit in here somewhere.

Reply:
Ah, yes. I think what's happened is that I looked at nullability 'solution wide'. I'll revert these ones

Reply:
I've reverted everything that is not directly related `System.Security.Cryptograph.Xml`. Sorry about that. I'm used to having solutions where everything in it is related and under the same folder as the solution.